pages/common/curl.md

```diff
@@ -5,32 +5,40 @@
 
 - Download a URL to a file:
 
-`curl "{{URL}}" -o {{filename}}`
+`curl {{http://example.com}} -o {{filename}}`
+
+- Download file using remote name:
+
+`curl -O {{http://example.com/filename}}`
+
+- Automatically continue/resume previous file transfer:
```

Comment:
Well, this PR results in more examples than what wget page has. So, I was thinking that since we usually try to keep it to 5,6 examples, it might be better to use the `-L` option rather than `-C`. Lets take a decision based on what others comment.